### PR TITLE
run cargo fmt, (not meant to be merged)

### PR DESCRIPTION
**Pull request by guanqun**
This is only for preview purpose.

### CODE REVIEW

File: build.rs [hunk 1]
```diff
@@ -21,7 +21,8 @@ const MIN_RUSTC_VERSION: &'static str = "1.15.1";
 fn main() {
 	let is = rustc_version::version().unwrap();
 	let required = MIN_RUSTC_VERSION.parse().unwrap();
-	assert!(is >= required, format!("
+	assert!(is >= required,
```
Comment (NikVolf) [inline build.rs]: already discussed, we will not align code this way, but

```rust
assert!(is >= required,
    other_para,
    final_param));
```

File: parity/blockchain.rs [hunk 2]
```diff
@@ -188,33 +188,13 @@ fn execute_import(cmd: ImportBlockchain) -> Result<(), String> {
...
 	// build client
-	let service = ClientService::start(
-		client_config,
-		&spec,
-		&client_path,
-		&snapshot_path,
-		&cmd.dirs.ipc_path(),
-		Arc::new(Miner::with_spec(&spec)),
-	).map_err(|e| format!("Client service error: {:?}", e))?;
+	let service = ClientService::start(client_config, &spec, &client_path, &snapshot_path, &cmd.dirs.ipc_path(), Arc::new(Miner::with_spec(&spec)))
```
Comment (NikVolf) [inline parity/blockchain.rs]: no way :)

File: build.rs (hunk 1 above)
Comment (NikVolf) [inline build.rs]: and almost everything below is about this

Comment (guanqun) [inline build.rs]: probably, the `rustfmt` is confused by the multiple lines in `format!`. Let me check if `rustfmt` has an option to format like this.

File: parity/blockchain.rs (hunk 2 above)
Comment (guanqun) [inline parity/blockchain.rs]: not sure which option controls the behavior...

**Pull request closed without merging**